### PR TITLE
FFC-163685: Obtain access token after redirection from DEFRA ID

### PR DESCRIPTION
**Pull request by marcin-mogiela**

### CODE REVIEW

File: app/auth/access-token/trusted-issuers/trusted-issuers.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const config = require('../../../config').authConfig
+
+module.exports = [
+  `https://${config.defraId.tenantName}.b2clogin.com/64c9d4f5-a560-4b65-9004-6d1e5ccee51d/v2.0/`
```
Comment (chrissmKainos) [inline app/auth/access-token/trusted-issuers/trusted-issuers.js:4]: Does the last part of this address need to be configurable?

Comment (liamwilson-kainos) [reply app/auth/access-token/trusted-issuers/trusted-issuers.js:4]: yeah definitely needs to be config and it should be available under existing env var `process.env.DEFRA_ID_TENANT`

File: app/session/index.js [hunk 2]
```diff
@@ -92,6 +94,14 @@ function getPkcecodes (request, key) {
   return get(request, entries.pkcecodes, key)
 }
 
+const setPerson = (request, key, value) => {
```
Comment (liamwilson-kainos) [inline app/session/index.js:97]: maybe want to refer to this as customer for consistency in other areas?

File: docker-compose.yaml [hunk 3]
```diff
@@ -37,5 +37,6 @@ services:
       DEFRA_ID_POLICY: ${DEFRA_ID_POLICY:-B2C_1A_SIGNUPSIGNINSFI}
       DEFRA_ID_REDIRECT_URI: ${DEFRA_ID_REDIRECT_URI:-http://localhost:3000/apply/signin-oidc}
       DEFRA_ID_CLIENT_ID: ${DEFRA_ID_CLIENT_ID:-83d2b160-74ce-4356-9709-3f8da7868e35}
-      DEFRA_ID_CLIENT_SECRET: ${DEFRA_ID_CLIENT_ID:-changeme}
+      DEFRA_ID_CLIENT_SECRET: ${DEFRA_ID_CLIENT_SECRET:-changeme}
       DEFRA_ID_SERVICE_ID: ${DEFRA_ID_SERVICE_ID:-2a672ee6-7750-ed11-bba3-000d3adf7047}
+      NODE_TLS_REJECT_UNAUTHORIZED: "0"
```
Comment (liamwilson-kainos) [inline docker-compose.yaml:42]: watch out for this, can we keep this just locally :)

**Review by chrissmKainos — APPROVED** [review]
Comments have been resolved, all looks good.

**Review by chrissmKainos — APPROVED** [review]

**Pull request merged**